pages/docs/specifications/v2.0.0.md

```diff
@@ -27,29 +29,72 @@ It means that the [application](#definitionsApplication) allows [consumers](#def
 
 **The AsyncAPI specification does not assume any kind of software topology, architecture or pattern.** Therefore, a server MAY be a message broker, a web server or any other kind of computer program capable of sending and/or receiving data. However, AsyncAPI offers a mechanism called "bindings" that aims to help with more specific information about the protocol and/or the topology.
 
-## Definitions {#definitions}
-
-#### Application {#definitionsApplication}
+## Table of Contents
```

Comment:
you can remove ToC, automation will also do it for future specs

Reply:
👍 nice! done :)